featuretools/tests/primitive_tests/test_transform_primitive.py

```diff
@@ -177,6 +179,64 @@ def test_is_quarter_start():
     np.testing.assert_array_equal(iqs_bools, correct_bools)
 
 
+def test_is_lunch_time_default():
+    is_lunch_time = IsLunchTime()
+    dates = pd.Series(
+        [
+            datetime(2022, 6, 26, 12, 12, 12),
+            datetime(2022, 6, 28, 12, 3, 4),
+            datetime(2022, 6, 28, 11, 3, 4),
+            np.nan,
+        ]
+    )
+    actual = is_lunch_time(dates)
+    expected = [True, True, False, False]
+    np.testing.assert_array_equal(actual, expected)
+
+
+def test_is_lunch_time_configurable():
+    is_lunch_time = IsLunchTime(14)
+    dates = pd.Series(
+        [
+            datetime(2022, 6, 26, 12, 12, 12),
+            datetime(2022, 6, 28, 14, 3, 4),
+            datetime(2022, 6, 28, 11, 3, 4),
+            np.nan,
+        ]
+    )
+    actual = is_lunch_time(dates)
+    expected = [False, True, False, False]
+    np.testing.assert_array_equal(actual, expected)
+
+
+def test_is_working_hours_standard_hours():
+    is_working_hours = IsWorkingHours()
+    dates = pd.Series(
+        [
+            datetime(2022, 6, 21, 16, 3, 3),
+            datetime(2019, 1, 3, 4, 4, 4),
+            datetime(2022, 1, 1, 12, 1, 2),
+        ]
```

Comment:
One last item - we should add a `nan` value here like we do in `IsLunchHour` just to confirm this primitive doesn't have issues with missing values.